editions/tw5.com/tiddlers/Breadcrumbs Macros.tid

```diff
@@ -0,0 +1,96 @@
+breadcrumbs: aa bb cc dd ee
```

Comment:
I don't like the pattern of making documentation tiddlers do double duty as example data. Examples should be made up of separate tiddlers that can be moved to another wiki independent of the documentation tiddlers.

Reply:
This field is bound to the 3rd example in the docs tiddler. It should only work there. 
It's intended to be used with "non existing tiddler titles", to show that it's not needed, that listed titles exist. 
It should also show, how the macro works with the default "start" parameter, which is `<<currentTiddler>>`. The first 2 examples use a "start" parameter which works with tiddlers that have to exist.